dali/pipeline/operators/reader/loader/loader.h

```diff
@@ -122,23 +122,22 @@ class Loader {
       for (int i = 0; i < initial_empty_size_; ++i) {
         LoadTarget* tensor = new LoadTarget();
```

Comment:
Why we are not using any smart pointers for that? In that case we would avoid calling delete directly on them.

Reply:
That's exactly what I said this morning :) But we are trying to fix the underlying problem and adding the shared pointer for testing now will just hide the problem.
Let's change it once we figure out the issue

Reply:
Understood

Reply:
done